maat/tests/real_world.rs

```diff
@@ -97,7 +100,7 @@ async fn set_porep_verifying_key<Keypair>(
     Keypair: subxt::tx::Signer<PolkaStorageConfig>,
 {
     let result = client
-        .set_porep_verifying_key(charlie, vk, true)
+        .set_porep_verifying_key(charlie, RegisteredSealProof::StackedDRG2KiBV1P1, vk, true)
```

Comment:
Nope, it should come as a param from the root of the test, plus this one won't work.
I think current default is 8MiB.